frontend/src/containers/TopicareaListing.tsx

```diff
@@ -153,7 +153,7 @@ function TopicareaListing() {
               <span>
                 <Button
                   onClick={createTopicArea}
-                >{`Create ${EnvConfig.topicAreaLabel.toLowerCase()}`}</Button>
+                >{`Create new ${settings.topicAreaLabels?.singular.toLowerCase()}`}</Button>
```

Comment:
If we make the `topicAreaLabels` property not optional, we don't need these conditional `?` accessors here. I think it looks cleaner without them and it is safe to assume that the property will always be there given that it has default values in both the `SettingsProvider` and the backend endpoint.